netket/graph/lattice.py

```diff
@@ -311,58 +257,51 @@ def __init__(
         self._pbc = self._clean_pbc(pbc, self._ndim)
 
         self._extent = _np.asarray(extent, dtype=int)
+        self._lattice_dims = _np.expand_dims(self._extent, 1) * self.basis_vectors
+        self._inv_dims = _np.linalg.inv(self._lattice_dims)
 
         self._point_group = point_group
 
-        sites, inside, self._basis_coord_to_site = create_sites(
+        # Generate sites
+        self._sites, self._basis_coords, self._positions = _create_sites(
```

Comment:
as i said above, i think here we're duplicating information... or are we not?

Reply:
We are, but `_sites` (or at least `Lattice.sites`) is needed for backwards compatibility. If you want to deprecate it, that could be done in a separate PR (but would just add more code with little benefit for the time being.)